src/Sidebar/Sidebar.scss

```diff
@@ -0,0 +1,33 @@
+.sidebar-container {
+    //font-size: 1.6rem;
+    background-color: #3D4836;
+    height: 100vh;
+    min-width: 250px;
+    color: white;
+    //position: fixed;
```

Comment:
I'm using grid on the dashboard -- thinking we can do the same for the other pages, which means we _should_ be able to use` position: fixed`.  I know the all horses page might have scrolling, and it would be nice if the sidebar stayed put.

Reply:
I've had better look making things responsive with flex, but I know it's totally doable with grid so probably either will work fine. I like the idea of a fixed sidebar.